lambda/custom/index.js

```diff
@@ -194,8 +283,9 @@ function startGame(newGame, handlerInput) {
   const currentQuestionIndex = 0;
   const spokenQuestion = Object.keys(translatedQuestions[gameQuestions[currentQuestionIndex]])[0];
   let repromptText = requestAttributes.t('TELL_QUESTION_MESSAGE', '1', spokenQuestion);
+
   for (let i = 0; i < ANSWER_COUNT; i += 1) {
-    repromptText += `${i + 1}. ${roundAnswers[i]}. `;
+    repromptText += `${i + 1}.  ${roundAnswers[i]}. `;
```

Comment:
whitespace.

Reply:
The whitespace is intended in order to Alexa to read the text properly. For example:
Instead of  1. answerX.2. Answer Y.
Will be: 1. answerX. 2. Answer Y.